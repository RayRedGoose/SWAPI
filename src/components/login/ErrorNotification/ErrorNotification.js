import React from 'react';
import './ErrorNotification.scss'

const ErrorNotification = () => {
  return (
    <p className="error-notification">Please, enter all data to continue!</p>
  )
}

export default ErrorNotification
