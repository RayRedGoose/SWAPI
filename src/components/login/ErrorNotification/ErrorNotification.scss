.error-notification {
  font-size: 24px;
  padding: 0.4em;
  margin: 0 40px;
  background-color: #BF2628;
  color: #fff;
}
