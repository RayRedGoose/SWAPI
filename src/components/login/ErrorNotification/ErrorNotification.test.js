import React from 'react'
import {shallow} from 'enzyme'
import ErrorNotification from './ErrorNotification'

it ('should match the snapshot with all data passed in correctly', () => {
  const error = shallow(<ErrorNotification />)
  expect(error).toMatchSnapshot()
})
