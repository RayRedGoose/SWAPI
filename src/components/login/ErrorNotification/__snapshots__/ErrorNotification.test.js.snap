// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot with all data passed in correctly 1`] = `
<p
  className="error-notification"
>
  Please, enter all data to continue!
</p>
`;
